services/util/src/services/downloads.js

```diff
@@ -52,6 +54,14 @@ module.exports = (DownloadsPersistence, CloudServices) => {
         uploaded.Location,
         uploaded.exp_date,
       );
+
+      fs.unlink(file.path, (err) => {
+        if (err) {
+          const Error = { message: 'Error by deleting temp file' };
+          throw Error;
```

Comment:
@erikasv please help me out to review how to handle error in this callback by deleting temp file. Let me know your opinion of this implementation.

Reply:
hmm, I don't think you should throw it, better just log the error. It's ok that is an async call and better not affect the rest of the flow, if it doesn't get deleted it's not a major problem (unless it happens a lot xD)

Reply:
Also I would remove the `by` in the message